### ISSUE TITLE
No mapping for the Unicode character exists in the target multi-byte code page

### ISSUE THREAD

**Issue by Lartza** [darkdks/KF2ServerTool#17]
**Describe the bug**
Weird/invalid characters in KFGame.ini break the tool

**To Reproduce**
1. Edit LinuxServer-KGame.ini motd to contain something like "B/B�G"
2. Run the tool with -info
3. Error: Failed to load /home/steam/kf2/KFGame/Config/Server1/LinuxServer-KFGame.ini: No mapping for the Unicode character exists in the target multi-byte code page

**Expected behavior**
Config files get read correctly and don't cause the tool to fail functioning

**Server:**
 - OS: Arch Linux
 - Version of the tool 1.3.5

**Additional context**
The characters got put there by kf2-magicked-admin and are characters some people use in their Steam names. KF2 Server doesn't seem to mind the characters

**Comment by darkdks** [comment]
This is a Encoding problem, to make it easy lease upload your LinuxServer-KGame.ini somewhere.

**Comment by Lartza** [comment]
Sure thing. Password removed, but the file still fails after saving with nano https://lartza.ltn.fi/LinuxServer-KFGame.ini

**Comment by darkdks** [comment]
I changed the Encoding for load and save to ANSI, that should now work with these special chars.
I think this encoding should be work fine since you are using it without problems.

**Comment by Lartza** [comment]
Fixed! Tried having the tool modify the files with the new version and everything was still fine.

This is without the tool touching the file
```
[steam@sauna kf2]$ file KFGame/Config/Server1/LinuxServer-KFEngine.ini
KFGame/Config/Server1/LinuxServer-KFEngine.ini: ASCII text
```
even though server starts with `[0000.93] Init: Character set: Unicode`
All files are as the dedicated server created them so no unicode it seems and ANSI is right. I guess only Tripwire knows :P

Thanks a lot for your tool and all these fixes! Didn't expect anything to be fixed this fast. It'll be super helpful for managing maps. Hopefully I won't find more ways to break it too soon, though I did have a tiny enhancement idea ;)
